### PR TITLE
Cleaned up aside formatting

### PR DESCRIPTION
**Pull request by pcantrell**
This addresses #325, at least in part.

I’m unsure how to update the integration specs. (My usual approach to git submodules is to hide until they go away.) Do I need to create a separate PR on the specs repo…? @jpsim, can you lend a hand with this if I make very sad puppy eyes?

### CODE REVIEW

**Comment by segiddins** [comment]
👍🏻 presuming everything ends up looking the same

**Comment by pcantrell** [comment]
Two visual changes:

1. Label comes out as “SEE ALSO” instead of “SEEALSO”
2. Tidied up ugly asymmetrical margins.

File: lib/jazzy/assets/css/jazzy.css.scss [hunk 1]
```diff
@@ -406,258 +361,29 @@ header {
...
-    -webkit-user-select: none;
-  }
-}
-
-div.note {
-  padding: 4px 0 2px 10px;
-  margin-bottom: 1em;
-  border-left: $aside_note_border;
+.aside, .Swift {
+  padding: 0.4em 0.8em;
```
Comment (jpsim) [inline lib/jazzy/assets/css/jazzy.css.scss]: why the change from px to em? I prefer the spacing from before.

**before**
image

**after**
image

**Comment by jpsim** [comment]
Other than the margin changes, :+1: thanks!

PS: I caught this as I was rebuilding the integration fixtures, and I'm happy to do it again once this is ready to go!

File: lib/jazzy/assets/css/jazzy.css.scss (hunk 1 above)
Comment (pcantrell) [inline lib/jazzy/assets/css/jazzy.css.scss]: Hmm, the “before” is weirdly asymmetric for me, and my change fixed it. _Both_ of your screenshots look wrong to my eyes. Will investigate when I have a free minute….

Comment (jpsim) [inline lib/jazzy/assets/css/jazzy.css.scss]: I agree that the "before" version isn't perfect, but better than what I generated from this PR.

Comment (pcantrell) [inline lib/jazzy/assets/css/jazzy.css.scss]: Right. What I’m saying is that they _both_ look quite different on my machine, before as well as after, and I need to figure out why — maybe local changes….

Comment (pcantrell) [inline lib/jazzy/assets/css/jazzy.css.scss]: Here’s what I see after the change:
screen shot 2015-10-28 at 6 26 19 pm
Note the symmetry above & below. Clearly not what you’re getting!

Comment (jpsim) [inline lib/jazzy/assets/css/jazzy.css.scss]: That looks quite nice! Maybe I'm generating this wrong? Seems unlikely though, since I ran `rake rebuild_integration_fixtures` and the new CSS classes were being resolved.

Comment (pcantrell) [inline lib/jazzy/assets/css/jazzy.css.scss]: Odds are good that it’s something about my hacked-up local setup. Investigating, and I’ll add a commit to the PR when I have it figured out.

Comment (pcantrell) [inline lib/jazzy/assets/css/jazzy.css.scss]: Aha! Neither your mistake nor mine. Sometimes the aside content is inside a <p> tag, and sometimes it isn’t. You were seeing a <p> case, and I was seeing a bare text case. Just pushed a fix to make it consistent.

**Comment by pcantrell** [comment]
OK, I think this is ready to go. Take a look at the new output and see how it looks to you now, @jpsim.

Do I need to have Xcode 6.x and 7.0 installed, as well as 7.1? I experimented with rebuilding the integration fixtures, but get errors about missing compilers for Swift 1.2 and 2.0.

**Comment by jpsim** [comment]
Looks good now! You can point the integration specs submodule to https://github.com/realm/jazzy-integration-specs/commit/fff44b9d7bffa9add4e55eddddd26174e220f247 now.

> Do I need to have Xcode 6.x and 7.0 installed, as well as 7.1? I experimented with rebuilding the integration fixtures, but get errors about missing compilers for Swift 1.2 and 2.0.

Currently compilers for Swift 1.2 and Swift 2.0 are required to run the full test suite.

**Comment by pcantrell** [comment]
Repointed integration specs and merged. First time doing that, so please LMK if I did anything wrong.

**Comment by jpsim** [comment]
Hurray!
